zap/src/main/java/org/parosproxy/paros/extension/manualrequest/http/impl/ManualHttpRequestEditorDialog.java

```diff
@@ -117,6 +120,15 @@ protected void initialize() {
         getFooterStatusBar().add(getLabelContentLength());
         getFooterStatusBar().addSeparator();
         getFooterStatusBar().add(getLabelTotalLength());
+
+        getWindowPanel()
+                .addComponentListener(
+                        new ComponentAdapter() {
+                            @Override
+                            public void componentShown(ComponentEvent ce) {
+                                sender.updateOptions();
```

Comment:
re: the earlier comment on `HttpPanelSender` this doesn't seem to be happening for some reason (or at least it wasn't on my Windows dev box).

Reply:
I think componentShown might not be appropriate for this case actually, will change.

Reply:
Which case is this addressing?

Reply:
@thc202 I had thought it might not update if the tracking state option was changed while the message sending panel was open, but that is not the case. It seems sufficient to add a call to the update function on construction of the window to address the initial concern.

Reply:
This change can be reverted then? (Also the changes in `HttpPanelSender` constructor and the new method`updateOptions`.)

Reply:
Yes. These changes are now reverted in the latest commit.